content/explanations/gamification/privileges.md

```diff
@@ -0,0 +1,26 @@
+---
+kind: explanation
+sidebar: ranks-honor
+prev: /explanations/gamification/honor/
+---
+
+# Privileges
+
+Users earn new privileges as their Honor grows. These can be divided in a couple of groups:
+
+- Authoring activities: a user is able to create new kata or translations.
+- Beta process: a user can vote on ranking and satisfaction of beta kata. Votes of more privileged users have more power than the others.
+- Content curation: a user can approve kata and translations after reviewing them, or edit kata to fix issues†.
+- Moderation: a user is able to mark comments as spoilers, unmark them, or mark issues as resolved.
+
+Privileges are earned by reaching a certain level of Honor. The exact required amounts to gain some privilege is listed in [this table](/references/gamification//privileges/).
+
+---
+
+**†** You can edit a kata created by others if:
+
+- You have contributed to the kata in the past (you do not need to have a coauthoring privilege in this case) **OR**
+- You have been granted coauthoring privilege **and** at least one of the following are true:
+  - The kata is in beta and waiting for approval
+  - The kata is in beta and the author have allowed contribution
+  - The kata is approved and unresolved issue/suggestion created over a week ago exists
```

Comment:
Move these under the table because it doesn't make sense on this page.

Reply:
Oh wait, I missed "or edit kata to fix issues†".

Reply:
I guess it's fine to omit the note on this page. Just also remove the dagger in "or edit kata to fix issues†".

Reply:
done